db.sql

```diff
@@ -1,7 +1,11 @@
-CREATE TABLE cliente (
-	CF int PRIMARY KEY,
-	Nome varchar(50) NOT NULL,
-	Cognome varchar(50) NOT NULL,
-	Tipo varchar(50) NOT NULL
+CREATE TABLE Forma(
```

Comment:
Spostate la creazione della tabella dopo la riga 20 e mettete il nome della tabella in minuscolo